src/main/kotlin/pl/edu/prz/kod/adapters/http/HttpHandler.kt

```diff
@@ -13,16 +16,25 @@ import java.util.*
 
 val base64Decoder by inject<Base64.Decoder>(Base64.Decoder::class.java)
 val executorOrchestrator by inject<ExecutorOrchestratorPort>(ExecutorOrchestratorPort::class.java)
+private val logger by inject<KLogger>(KLogger::class.java)
```

Comment:
I think it would be better to create logger instance here instead of injecting it. Injected instance will show in logs `ApplicationModule`, while the one created here would show `HttpHandler`, which is more appropriate in my opinion.

```suggestion
private val logger = KotlinLogging.logger {}
```

Also, the two fields (or whatever it's called in kotlin) above could be private I guess.